### PR TITLE
Eric style updates

### PR DESCRIPTION
**Pull request by EricBenjamin**
New and improved styling that was lost in my previous merge-conflict. Also includes: new button class and new toggle bar class. Bonus: styling for Ricky's soon-to-be-depricated drop down list.

### CODE REVIEW

**Comment by tdumitrescu** [comment]
@EricBenjamin can you remove the "bonus" first? Considering that https://github.com/mixpanel/mixpanel-common/pull/72/commits/406dbf8d4dde87452d44c046103298ccb028ff8e will likely not make it into master, I'd like for this branch to be mergeable before reviewing.

**Comment by EricBenjamin** [comment]
Ok @tdumitrescu I've removed all the menu stuff, but didn't rebase / change branches.

**Comment by tdumitrescu** [comment]
Well the branch won't be mergeable as long as https://github.com/mixpanel/mixpanel-common/pull/72/commits/406dbf8d4dde87452d44c046103298ccb028ff8e is still in it. If you'd like I can remove it for you and force-push this branch.

**Comment by EricBenjamin** [comment]
@tdumitrescu that would be great, sorry :( git rookie here.

**Comment by tdumitrescu** [comment]
aight homes I got rid of the menu business, so should be ready for some rounds of review and cleanup...

File: examples/style-guide/index.jade [hunk 1]
```diff
@@ -61,23 +61,25 @@
...
-      mp-button.mp-button-secondary(attributes={icon: 'logout'}) Exit
-      mp-button.mp-button-primary(attributes={icon: 'caret-down'}) Menu closed
-      mp-button.mp-button-primary(attributes={icon: 'plus'}) Create
-      mp-button.mp-button-primary.mp-button-active(attributes={icon: 'caret-up'}) Menu open
-      mp-button.mp-button-red(attributes={icon: 'trashcan'}) Delete
-      mp-button.mp-button-primary No icon
+      mp-button.mp-button-secondary.mp-button-with-icon(attributes={icon: 'x'}) Close
+      mp-button.mp-button-secondary.mp-button-with-icon(attributes={icon: 'logout'}) Exit
+      mp-button.mp-button-primary.mp-button-with-icon(attributes={icon: 'plus'}) Create
+      //- mp-button.mp-button-primary.mp-button-with-icon(attributes={icon: 'caret-down'}) Menu closed
```
Comment (jbwyme) [inline examples/style-guide/index.jade]: just remove these instead of commenting them out

File: lib/components/button/index.styl [hunk 2]
```diff
@@ -108,7 +107,7 @@ icon-size(size=none, pos-left=0, pos-top=0)
     &.secondary, &.grey
       background-image linear-gradient(0deg, grey-50 0%, white 100%)
       border-color grey-300
-      box-shadow 0px 1px 1px 0px rgba(0,0,0,0.1)
+      // box-shadow 0px 1px 1px 0px rgba(0,0,0,0.1)
```
Comment (jbwyme) [inline lib/components/button/index.styl]: remove

File: lib/components/toggle/index.styl [hunk 3]
```diff
@@ -20,20 +22,22 @@
...
     text-transform uppercase
     transition background-color 0.2s
     &.mp-toggle-selected
       background-color #fff
+      box-shadow 0 1px 0 0 (rgba(0,0,0,0.05))
     &:not(.mp-toggle-selected)
       cursor pointer
       &:hover
-        color blue-300
+        color blue-300
```
Comment (jbwyme) [inline lib/components/toggle/index.styl]: remove trailing space

File: lib/components/toggle/index.styl [hunk 4]
```diff
@@ -42,3 +46,6 @@
         color #fff
       &:not(.mp-toggle-selected):hover
         color #fff
+
+.large-toggle
+    height 40px
```
Comment (jbwyme) [inline lib/components/toggle/index.styl]: add new line

File: lib/components/tooltip/index.styl [hunk 5]
```diff
@@ -34,3 +35,5 @@ json('../../../stylesheets/colors.json')
         position absolute
         top 100%
         width 0
+
+
```
Comment (jbwyme) [inline lib/components/tooltip/index.styl]: remove

**Comment by EricBenjamin** [comment]
BOOM. done.

File: lib/components/toggle/index.styl (hunk 4 above)
Comment (EricBenjamin) [inline lib/components/toggle/index.styl]: I *think* this is done, but github is not calling it "outdated" so yeah sup with that.

**Pull request merged**